### DISCUSSION TITLE
App.Current.Exit() fails on a Skia.Gtk project.

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#9992] [Q&A] [answered]
I added a method in one of my view models to allow the user to exit the application.

It works on Windows but fails in Skia.Gtk projects returning this error:

```
Object reference not set to an instance of an object.
   at Uno.UI.Runtime.Skia.GtkApplicationExtension.Exit()
```

I've added a minimal example.

[UnoTestExit.zip](https://github.com/unoplatform/uno/files/9657920/UnoTestExit.zip)

**Comment by jeromelaban** [comment]
Thanks for the report!

**Comment by MartinZikmund** [comment, marked as answer]
@mcNets Will be fixed by #10001!
